test/mCODE.test.js

```diff
@@ -1,21 +1,21 @@
-const { loadELM, loadJSONFixture, loadValueSets } = require('./helpers/fixtureLoader');
-const { mapValueSets } = require('./helpers/valueSetMapper');
-const { execute } = require('./helpers/execution');
+const { loadELM, loadJSONFixture, loadValueSets } = require('../testing-harness/fixtureLoader');
+const { mapValueSets } = require('../testing-harness/valueSetMapper');
+const { execute } = require('../testing-harness/execution');
 
 let executionResults;
 beforeAll(() => {
   // Set up necessary data for cql-execution
-  const valueSets = loadValueSets();
+  const valueSets = loadValueSets('../valuesets');
```

Comment:
It looks like the `loadValueSets` function doesn't take in any arguments. Were you planning to make that change? If not, I think we can revert this line to just `loadValueSets()`.